windows-rdp/main.tf

```diff
@@ -9,6 +9,15 @@ terraform {
   }
 }
 
+variable "share" {
+  type    = string
+  default = "owner"
+  validation {
```

Comment:
We can still add a `description` as they are rendered in registry.coder.com/modules variables tab.

Reply:
Would it be good to add that value for the `share` variables in the other modules, too?
Definitely on board with more documentation, but I don't think we're doing this anywhere just yet

Reply:
Cool cool 😎 I can make an issue for adding descriptions to these fields/generally formalizing documentation for modules and do it in a separate PR.

Reply:
Thank you 😊